develop/parachain-devs/get-started/polkadot-sdk/get-started-psdk.md

```diff
@@ -0,0 +1,102 @@
+---
+
+!!! danger "Not a production ready guide."
+    This section is under construction and moving!
+
+    Agile coretime is only for the Kusama and testnet networks at the moment, and is not yet deployed on
+    Polkadot. These guides are **not** production ready due to the moving nature of these features.
+
+## Using the Polkadot SDK
+
+The Polkadot SDK monorepo is composed of **three** repositories:
+
+- [**Polkadot**](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html#polkadot){target=_blank} -
+  For a time, this included both the client implementation and runtime, until the runtime was moved
+  to the [Polkadot Fellows organization](https://github.com/polkadot-fellows/runtimes){target=_blank}.
+- [**Substrate**](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html#substrate){target=_blank} -
+  The underlying, core primitives and libraries for building blockchains (any blockchain, not just
+  one for Polkadot). Polkadot itself is built with Substrate.
+- [**Cumulus**](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html#cumulus){target=_blank} -
+  A set of libraries and tools which pertain specifically with connecting blockchains to Polkadot,
+  known as parachains.
+
+For an in-depth dive into the monorepo, the [Polkadot SDK Rust documentation](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html){target=_blank} is highly recommended
+
+<!-- TODO: Glossary link ParaId -->
+
+## Zero to Hero with The Polkadot SDK
+
+The process of using the Polkadot SDK to deploying a parachain on a [core](https://wiki.polkadot.network/docs/learn-agile-coretime#core){target=_blank} can be summarized as follows. It is also possible to build standalone blockchains with the Polkadot SDK:
+
+1. **Reserving** a `ParaId`, where the runtime and genesis state will be uploaded
+2. **Compiling** the runtime (written in Rust) to a WebAssembly blob, thereby defining how state transitions from one state to the next. This runtime is created using the Polkadot SDK
+3. **Generating** a chain spec that is viable and ready to be deployed as a live, working parachain
+4. **Generating** the genesis state and wasm
+5. **Obtaining** a core through a [Coretime marketplace](https://wiki.polkadot.network/docs/learn-guides-coretime-marketplaces){target=_blank}
+6. **Assigning** that core to a `ParaId`
+7. **Provisioning** at least one honest, synced collator is available for the task to begin producing blocks
+
+!!!info "What is a task?"
+    A task refers to a process utilizing Polkadot's compute. This could be a parachain or any other computational process, provided that it adheres to the Polkadot protocol. The full definition can be found on the [Polkadot Wiki](https://wiki.polkadot.network/docs/learn-agile-coretime#task).
```

Comment:
```suggestion
    A [task](https://wiki.polkadot.network/docs/learn-agile-coretime#task){target=\_blank} refers to a process utilizing Polkadot's compute. This could be a parachain or any other computational process, provided that it adheres to the Polkadot protocol.
```